src/app/components/search/SearchWithCreate.tsx

```diff
@@ -3,17 +3,21 @@ import { Box, Button, FormControl, TextInput } from '@primer/react'
 import { Stack } from '@primer/react/lib-esm/Stack'
 import { ChangeEvent } from 'react'
 
-interface MirrorSearchProps {
+interface SearchWithCreateProps {
+  placeholder: string
+  createButtonText: string
   searchValue: string
   setSearchValue: (value: string) => void
   openCreateDialog: () => void
 }
 
-export const MirrorSearch = ({
+export const SearchWithCreate = ({
+  placeholder,
+  createButtonText: createButtonLabel,
```

Comment:
Is the rename needed here? Let's stick to whatever the prop name is

Reply:
Ah apologies, I mean the `createButtonText` rename

Reply:
Oh, I missed that.  Yeah it doesn't need the rename.